testing/report11.html

```diff
@@ -7732,21 +7729,41 @@ <h1 id="h_appendix"><a id="appendix" name="appendix">Appendix</a></h1>
 		
 <h2 id="h_ack"><a id="ack" name="ack">Acknowledgements</a></h2>
 <p>
-  The Web of Things Working Group would like to acknowledge the contributions to the making of this document from the following individuals in various capacities.</p>
-<p>
-</p><ul>
+The Web of Things Working Group would like to acknowledge the contributions to 
+the making of this document from the following individuals in various capacities,
+including contributions to past testing inputs which were used for compatibility testing:</p>
+<ul>
```

Comment:
`Cristiano Aguzzi (Invited Expert)` is missing

Reply:
@chachamimm let us know if you have done contributions to the implementation report.